phpstan-baseline.neon

```diff
@@ -6441,7 +6441,7 @@ parameters:
 			path: src/lib/Form/Type/Content/CustomUrl/CustomUrlRemoveType.php
 
 		-
-			message: "#^Call to an undefined method Symfony\\\\Component\\\\Form\\\\ChoiceList\\\\Loader\\\\ChoiceLoaderInterface\\:\\:setRestrictedContentTypeIds\\(\\)\\.$#"
+			message: "#^Call to an undefined method Symfony\\\\Component\\\\Form\\\\ChoiceList\\\\Loader\\\\ChoiceLoaderInterface\\:\\:setTargetLocation\\(\\)\\.$#"
```

Comment:
maybe we shouldnt type to generic interface there if we expect two specific methods calls there.